flights/ncric_v5alpr_historical_flight.yaml

```diff
@@ -111,16 +109,16 @@ entityDefinitions:
         transforms:
         - !<transforms.ConcatTransform>
           columns: ["gpsLat","gpsLng"]
-      # ol.locationcoordinates:
-      #   type: "ol.locationcoordinates"
-      #   transforms:
-      #   - !<transforms.GeographyPointTransform>
-      #     latTransforms:
-      #     - !<transforms.ColumnTransform>
-      #       column: "gpsLat"
-      #     lonTransforms:
-      #     - !<transforms.ColumnTransform>
-      #       column: "gpsLng" 
+      ol.locationcoordinates:
+        type: "ol.locationcoordinates"
+        transforms:
+        - !<transforms.GeographyPointTransform>
+          latTransforms:
+          - !<transforms.ColumnTransform>
```

Comment:
All instances of `!<transforms.ColumnTransform>` will need to be changed to `- !<com.openlattice.shuttle.transforms.ColumnTransform>` whenever shuttle changes are pulled

Reply:
ok, good to know!